### PR TITLE
Use community version of RN webview

### LINKED ISSUE #35: Warning: WebView has been extracted from react-native core
As of React Native 0.58, WebView has been decoupled and is now published from https://github.com/react-native-community/react-native-webview

Any plan to use `react-native-webview` for WebView?

### PR DESCRIPTION
**Pull request by karanjthakkar**
Fixes #35. I think this should potentially be released in major version to avoid breaking apps still on older RN versions.

### CODE REVIEW

**Comment by henrymoulton** [comment]
@karanjthakkar any idea if you have plans to merge this?

**Comment by karanjthakkar** [comment]
I'm waiting for someone to review and merge this.

File: package.json [hunk 1]
```diff
@@ -10,7 +10,8 @@
   "license": "MIT",
   "peerDependencies": {
     "react": ">=15",
-    "react-native": ">=0.48"
+    "react-native": ">=0.48",
```
Comment (seekshiva) [inline package.json]: Will this work alongside old versions of RN?

Comment (seekshiva) [reply package.json]: react-native-webview's peer dependency says `>=0.57 <0.59`:
https://github.com/react-native-community/react-native-webview/blob/master/package.json#L25

Can you update this in the PR? I'll merge it.

Comment (karanjthakkar) [reply package.json]: Done

**Comment by seekshiva** [comment]
v2.0.0 is out now

**Pull request merged**